tests/python/gaia-ui-tests/gaiatest/apps/marketplace/app.py

```diff
@@ -70,6 +70,10 @@ def tap_install_button(self):
         self.root_element.find_element(*self._install_button_locator).tap()
         self.marionette.switch_to_frame()
 
+    def tap_open_app_button(self):
+        if self.root_element.find_element(*self._install_button_locator).text == 'Open app':
+            self.root_element.find_element(*self._install_button_locator).tap()
```

Comment:
This doesn't seem like a good idea to me, the if call.
What happens when there is no 'Open app'?